source/docs/software/commandbased/commands-v3/binding-commands-to-triggers-v3.rst

```diff
@@ -0,0 +1,425 @@
+# Binding Commands to Triggers
+
+Triggers in Commands v3 work similarly to Commands v2: they bind commands to conditions like button presses or sensor states. This article covers the v3-specific API and patterns.
+
+## What Are Triggers?
+
+A ``Trigger`` is an object that represents a boolean condition (like a button being pressed). You can bind commands to trigger state changes using methods like ``onTrue()``, ``whileTrue()``, and ``onFalse()``.
+
+.. code-block:: java
+
+   import org.wpilib.commands3.Trigger;
+   import org.wpilib.commands3.button.CommandXboxController;
+
+   // Button trigger
+   CommandXboxController controller = new CommandXboxController(0);
+   Trigger aButton = controller.a();
+
+   // Custom condition trigger
+   Trigger sensorTriggered = new Trigger(() -> sensor.getValue() > threshold);
+
+## Creating Triggers
+
+### From Controller Buttons
+
+The ``CommandXboxController`` class provides triggers for all buttons:
```

Comment:
We should mention that other controller types are also supported, and that the xbox controller was only chosen for sake of example